tests/conda_env/test_cli.py

```diff
@@ -203,12 +218,24 @@ def test_update(self):
         self.assertNotEqual(len(parsed), 0)
 
     def test_name(self):
+        """
         # smoke test for gh-254
+        Test that --name can overide the `name` key inside an environment.yml
+        """
         create_env(environment_1)
+        env_name = 'smoke-gh-254'
         try:
-            run_env_command(Commands.ENV_CREATE, test_env_name_1, "create")
+            run_env_command(Commands.ENV_CREATE, 'environment.yml', "-n",
+                            env_name)
         except Exception as e:
-            self.assertIsInstance(e, EnvironmentFileNotFound, str(e))
+            print(e)
+
+        o, e = run_conda_command(Commands.INFO, None, "--json")
+
+        parsed = json.loads(o)
+        self.assertNotEqual(
+            len([env for env in parsed['envs'] if env.endswith(env_name)]), 0
+        )
```

Comment:
I am not sure what this test was actually doing, but I updated it to check @kalefranz mentioned the issue was.